distribution/docs/src/main/resources/_contents/_configuring/configuring-from-configuration-files-contents.adoc

```diff
@@ -0,0 +1,746 @@
+For example, \http://${branding}_HOST:8993/services/catalog/sources/ddf.distribution/2f5db9e5131444279a1293c541c106cd?transform=resource&user=1w1qlo79j6tscii19jszwp9s2i55 notifications contain the following information:
+
+.Notifications Parameters
+[cols="1,4,1" options="header"]
+|===
+
+|Parameter Name
+|Description
+|Required by ${branding} Standard UI
+
+|`application`
+|"Downloads" for resource retrieval.
+This is used as a "type" or category of messages.
+|Yes
+
+|`title`
+|Resource/file name for resource retrieval.
+|Yes
+
+|`message`
+|Human-readable message containing status and a more detailed message.
+|Yes
+
+|`timestamp`
+|Timestamp in milliseconds of when event occurs.
+|Yes
+
+|`user`
+|CometD Session ID or unique User ID.
+|Yes
+
+|`status`
+|Status of event.
+|No
+
+|`option`
+|Resource retrieval option.
+|No
+
+|`bytes`
+|Number of bytes transmitted.
+|No
+
+|===
+
+===== Receive Notifications
+
+* If interested in retrieve resource notifications, a client must subscribe to the CometD `channel/${ddf-branding-lowercase}/notification/catalog/downloads`.
+* If interested in all notification types, a client must subscribe to the CometD `channel/${ddf-branding-lowercase}/notification/**`
+* A client will only receive notifications for resources they have requested.
+* ${ddf-branding} Standard UI is subscribed to all notifications of interest to that `user/browser session: /${ddf-branding-lowercase}/notification/**`
+
+===== Publish Notifications
+
+Any application running in ${branding} can publish notifications that can be viewed by the ${ddf-branding} Standard UI or received by another notifications client.
+
+. Set a properties map containing entries for each of the parameters listed in the table above.
+. Set the OSGi event topic to `${ddf-branding-lowercase}/notification/<application-name>/<notification-type>`. Notice that there is no preceding slash on an OSGi event topic name, but there is one on the CometD channel name.
+The OSGi event topic corresponds to the CometD channel this is published on.
+. Post the notification to the OSGi event defined in the previous step.
+
+.Example for Publishing Notification
+[source,java,linenums]
+----
+Dictionary <String, Object> properties = new Hashtable<String, Object>();
+properties.put("application", "Downloads");
+properties.put("title", resourceResponse.getResource().getName());
+Long sysTimeMillis = System.currentTimeMillis();
+properties.put("message", generateMessage(status, resourceResponse.getResource().getName(), bytes, sysTimeMillis, detail));
+properties.put("user", getProperty(resourceResponse, USER));
+properties.put("status", "Completed");
+properties.put("bytes", 1024);
+properties.put("timestamp", sysTimeMillis);
+
+Event event = new Event("${ddf-branding-lowercase}/notification/catalog/downloads", properties);
+
+eventAdmin.postEvent(event);
+----
+
+==== Configuring Solr Catalog Provider Data Directory
```

Comment:
So far, all system.properties configurations are documented in section 3.3.1. Should we be consistent and have one section per topic, regardless of where the configuration properties are located, or have a section per configuration file and have all the property descriptions in that same section? If we go with the latter then this section should be merged with section 3.3.1.